### PR TITLE
Fetch & parse oh/state for OH

### LINKED ISSUE #488: fetch oh/state
[learn our pipeline: fetch](https://github.com/CAVaccineInventory/vaccine-feed-ingest/wiki/Runner-pipeline-stages#fetch)

Fetch data from: https://coronavirus.ohio.gov/wps/portal/gov/covid-19/dashboards/covid-19-vaccine/covid-19-vaccine-provider-dashboard

Put your script in a file named: `oh/state/fetch.sh` (or `fetch.sh`, etc.)

Store it (without processing) in a new file created in the directory passed as the first argument (`sys.argv[1]`).

Check the wiki to learn more about the purpose of the fetch stage and how to get set up for development!

### Tips

1. While working on your code, run it at any point:
    ```sh
    poetry run vaccine-feed-ingest fetch oh/state
    ```

### PR DESCRIPTION
**Pull request by cdjj**
| Key Details |
|-|
| Resolves #488, #508 |
State: OH |
Site: state |

## Data sample
```json
{"id": "aacs_superior_intermediate_gymnasium2308_wade_ave_provided_by_ashtabula_city_health_departmentashtabula_oh_44004", "contact": {"website": "http://www.cityofashtabula.com/health-department", "phone": "(440) 992-7122"}, "name": "AACS  Superior Intermediate Gymnasium", "address": "2308 Wade Ave Provided by Ashtabula City Health Department\nAshtabula, OH 44004"}
{"id": "acme_pharmacy3235_manchester_rd_akron_oh_44319", "contact": {"website": "http://www.acmestores.com", "phone": "(330) 644-0350"}, "name": "Acme Pharmacy", "address": "3235 Manchester Rd. \nAkron, OH 44319"}
{"id": "acme_pharmacy3979_medina_rd_akron_oh_44333", "contact": {"website": "http://www.acmestores.com", "phone": "(330) 666-3300"}, "name": "Acme Pharmacy", "address": "3979 Medina Rd. \nAkron, OH 44333"}
{"id": "adams_county_regional_medical_center230_medical_center_drive_seaman_oh_45679", "contact": {"website": "http://www.acrmc.com", "phone": "(937) 386-3400"}, "name": "Adams County Regional Medical Center", "address": "230 Medical Center Drive \nSeaman, OH 45679"}
```

## Before Opening a PR
- [X] I tested this using the CLI (e.g., `poetry run vaccine-feed-ingest <state>/<site>`)
- [X] I ran auto-formatting: `poetry run tox -e lint-fix`

### CODE REVIEW

**Review by eliblock — CHANGES_REQUESTED** [review]
Thank you for getting this rolling! I _believe_ that we might be dropping 50% of the data available to the parser - could you check?

File: vaccine_feed_ingest/runners/oh/state/parse.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    transposed_data = map(
+        dict,
+        itertools.starmap(
+            zip, zip(itertools.repeat(data_dict.keys()), zip(*data_dict.values()))
+        ),
+    )
+    # Data contains at least one bad value; filter it out. See https://github.com/CAVaccineInventory/vaccine-feed-ingest/issues/621
+    filtered_transposed_data = (
+        row for row in transposed_data  # if row["Site-value"] != "%null%"
+    )
```
[the remark this answers is not in the document]
Comment (cdjj) [reply vaccine_feed_ingest/runners/oh/state/parse.py:64]: Yep, that was there to get things rolling while I figured out the structure of the data.  Thanks for catching!

File: vaccine_feed_ingest/runners/oh/state/parse.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    # Adjacent rows are actually duplicates; some have map, some have website. Combine into one.
+    doubled_filtered_transposed_data = zip(
+        filtered_transposed_data, filtered_transposed_data
+    )
```
[the remark this answers is not in the document]
Comment (cdjj) [reply vaccine_feed_ingest/runners/oh/state/parse.py]: This was left over from the la/tableau/parse.py that I used to start this from; you're entirely correct that it's unneeded.  Thanks!

**Comment by cdjj** [comment]
Yep, it was!  Now getting 1200+ results, which seems about right, eyeballing the site.

**Review by eliblock — APPROVED** [review]
Nice work with this parser (the tableau parsers are exciting in general)!

**Pull request merged**